e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js

```diff
@@ -180,7 +180,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
 
     // GUEST PERMISSIONS DON'T EXIST ON TEAM OVERRIDE SCHEMES SO GUEST PORTION NOT IMPLEMENTED!
     // ONLY THE MEMBERS PORTION OF THIS TEST IS IMPLEMENTED
-    it('Post Reactions option removed for Guests & Members in Team Override Scheme', () => {
+    it('MM-T1546 Post Reactions option removed for Guests & Members in Team Override Scheme', () => {
```

Comment:
Could you please check this again? There are 4 steps in TM4J which I could only match step 4 but not the rest of the steps.

Reply:
Updated key to MM-T1546_4

Reply:
Thanks! Let's deal with steps 1-3 separately.